types/remarkable/tsconfig.json

```diff
@@ -7,7 +7,7 @@
         "noImplicitAny": true,
         "noImplicitThis": true,
         "strictNullChecks": true,
-        "strictFunctionTypes": true,
+        "strictFunctionTypes": false,
```

Comment:
why strict need to be off?

Reply:
This strict option need to be removed to ensure a non breaking change.

In `type Rule`, I've changed the typing to rely on generics but also made sure that it's still remain usable without it.